led_interface.py

```diff
@@ -1,21 +1,22 @@
 import collections
+from chess_position import ChessPosition
 
 class LedInterface():
 
     def __init__(self):
-        pass
+        self.chess_pos = ChessPosition(self, col, row)
 
-    def turn_on_led(self, position):
```

Comment:
Keep this line, all the methods should take a single `position` argument. This `position` argument will be a `ChessPosition` on the board.